test/Npgsql.PluginTests/NodaTimeTests.cs

```diff
@@ -139,37 +139,41 @@ public void TimestampTz()
                 var offsetDateTime = localZonedDateTime.ToOffsetDateTime();
                 var dateTimeOffset = offsetDateTime.ToDateTimeOffset();
+                var localDateTime = dateTimeOffset.LocalDateTime;
 
-                conn.ExecuteNonQuery("CREATE TEMP TABLE data (d1 TIMESTAMPTZ, d2 TIMESTAMPTZ, d3 TIMESTAMPTZ, d4 TIMESTAMPTZ, d5 TIMESTAMPTZ)");
+                conn.ExecuteNonQuery("CREATE TEMP TABLE data (d1 TIMESTAMPTZ, d2 TIMESTAMPTZ, d3 TIMESTAMPTZ, d4 TIMESTAMPTZ, d5 TIMESTAMPTZ, d6 TIMESTAMPTZ)");
 
-                using (var cmd = new NpgsqlCommand("INSERT INTO data VALUES (@p1, @p2, @p3, @p4, @p5)", conn))
+                using (var cmd = new NpgsqlCommand("INSERT INTO data VALUES (@p1, @p2, @p3, @p4, @p5, @p6)", conn))
                 {
                     cmd.Parameters.Add(new NpgsqlParameter("p1", NpgsqlDbType.TimestampTz) { Value = instant });
                     cmd.Parameters.Add(new NpgsqlParameter { ParameterName = "p2", Value = utcZonedDateTime });
                     cmd.Parameters.Add(new NpgsqlParameter { ParameterName = "p3", Value = localZonedDateTime });
                     cmd.Parameters.Add(new NpgsqlParameter { ParameterName = "p4", Value = offsetDateTime });
                     cmd.Parameters.Add(new NpgsqlParameter { ParameterName = "p5", Value = dateTimeOffset });
+                    cmd.Parameters.Add(new NpgsqlParameter { ParameterName = "p6", Value = localDateTime });
                     cmd.ExecuteNonQuery();
                 }
 
-                using (var cmd = new NpgsqlCommand("SELECT d1::TEXT, d2::TEXT, d3::TEXT, d4::TEXT, d5::TEXT FROM data", conn))
+                using (var cmd = new NpgsqlCommand("SELECT d1::TEXT, d2::TEXT, d3::TEXT, d4::TEXT, d5::TEXT, d6::TEXT FROM data", conn))
                 using (var reader = cmd.ExecuteReader())
                 {
                     reader.Read();
                     // When converting timestamptz as a string as we're doing here, PostgreSQL automatically converts
                     // it to the session timezone
-                    for (var i = 0; i < reader.FieldCount; i++)
+                    for (var i = 0; i < reader.FieldCount - 1; i++)
                         Assert.That(reader.GetValue(i), Is.EqualTo(
                             localZonedDateTime.ToString("uuuu'-'MM'-'dd' 'HH':'mm':'ss'.'fff", CultureInfo.InvariantCulture) + "-04")
                         );
+
+                    Assert.That(reader.GetValue(5), Is.EqualTo(localDateTime.ToString("yyyy-MM-dd HH:mm:ss.FFF-04", CultureInfo.InvariantCulture)));
                 }
 
                 using (var cmd = new NpgsqlCommand("SELECT * FROM data", conn))
                 using (var reader = cmd.ExecuteReader())
                 {
                     reader.Read();
 
-                    for (var i = 0; i < reader.FieldCount; i++)
+                    for (var i = 0; i < reader.FieldCount - 1; i++)
```

Comment:
What's the reason to ignore the last field?

Reply:
Lazy quickfix. See updated MR recent changes.